docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+
+Working Groups should be tracking features for the release, as well as the documentation status of each feature as it’s being developed so that the documentation team can keep track of the documentation work that needs to be done.
+
+## Terminology
+
+### Shadowing
+
+A person that has worked closely with the Release Manager in the past. Based on that interaction they should be able to handle tasks that the Release Manager may not be able to cover and share knowledge about the release process.
+
+### git revision
+
+A git revision is either a git commit, branch, or tag. Ideally people should be providing the manifests repo with tags, to update the manifests from.
+
+
+### Release Candidate (RC)
+
+The Release Candidate (RC) in the manifests repo is always a git **tag**.
+
+The manifests repo will be following the release process below:
+
+1. There will be a **vX.Y-branch** branch, in which the Manifests WG leads will be cherry picking commits on top
+2. Manifests leads will be creating new tags for different RCs, that will be pointing to commits of that release branch
+
+
+## Timeline
+
+
+### Preparation
+
+- [ ] Select a release team
+- [ ] Working groups broadly think about features **with priorities** they want to land for that cycle, have internal discussions, perhaps groom a backlog from previous cycle, get issues triaged, etc.
+- [ ] Ensure members of the Release Team are part of the [release-team](https://github.com/kubeflow/internal-acls/blob/54a454b92e34a98f74a8dc07216dd578f6fa40d5/github-orgs/kubeflow/org.yaml#L950) group
+- [ ] Establish a regular release team meeting as appropriate on the schedule, start off slowly, but meet more often towards the end.
+- [ ] Publish draft schedule to kubeflow-discuss, with actual dates
+- [ ] Get lazy consensus on the release schedule from the WG leads
+- [ ] Ensure schedule also accounts for the point releases AFTER the major release
+
+Criteria for timeline that the team needs to consider
+- Holidays around the world that coincide with members of the release team, WG representatives, and distro representatives.
+- Enterprise budgeting/approval lifecycle. (aka users have their own usage and purchase requirements and deadlines)
+- Kubecon dates - let’s not hard block on events, but keep them in mind since we know community members might get doublebooked.
+- Associated events (aka. AI Day at Kubecon, Tensorflow events) - we want to keep them in mind.
+
+**Success Criteria:** Release team selected, schedule sent to kubeflow-discuss, all release team members have the proper permissions and are meeting regularly.
+
+
+### Development (10 weeks)
+
+Normal development in the different WGs and in the <https://github.com/kubeflow/manifests> repo.
+
+**Success Criteria:**
+* (Optional but encouraged): Issues tracking new features should also provide information on whether the docs should be updated for that feature
```

Comment:
I'll also propose a KanBan board there instead of just an issue.

Reply:
Actually I had read the comment wrong.

The important part here is that issues for features or bugs should also have a comment on whether the docs should be updated. The Docs Lead can then keep track of the changes that need to be done based on the information from the existing issues

Reply:
My constant call for a KanBan in the comments might be irrelevant if we get into the new GitHub issues beta. Either way, I'll put in the work to implement either one of them and coordinate the implementation.

Reply:
It's not irrelevant, and it's really nice that you are eager and try to find ways to better automate and enhance the release! Pointing these things out gives us good context to keep on iterating and improving what we have.

The reason that I want us to focus on the **what** rather than the **how** right now is to avoid locking ourselves into a direction over the other. We can be more loose at this point and once we find out what workflows work for us we can then further extend the handbook with what we've learned and seen